nncf/common/graph/operator_metatypes.py

```diff
@@ -116,6 +117,11 @@ def get_operator_metatype_by_op_name(self, op_name: str) -> Type[OperatorMetatyp
         :return: The operator metatype.
         """
         if op_name not in self._op_name_to_op_meta_dict:
+            nncf_logger.warning(
+                f'The node with name {op_name} was mapped to UnknownMetatype,'
+                ' which means that there was not registered such NNCF metatype. '
+                'It could lead to not the best optimized model. '
+                'Please, Inform the NNCF developers about this message.')
```

Comment:
Do we really need this warning? In this case, we should add a meta type for each operation in the framework.
@alexsu52 @vshampor

Reply:
Will move the warning to debug shortly